_includes/integrations/react-tech-ref.md

```diff
@@ -40,6 +41,27 @@ or with Yarn
 $ yarn add @tinymce/tinymce-react
 ```
 
+### Using TinyMCE React integration in a Bootstrap dialog
+
+To use TinyMCE React instances inside Bootstrap UI dialogs, add the following React effect to a component that renders with the editor:
+
+```jsx
+  useEffect(() => {
+    const handler = (e) => {
+      if (
+        $(e.target).closest(
+          ".tox-tinymce-aux, .moxman-window, .tam-assetmanager-root"
+        ).length
+      ) {
+        e.stopImmediatePropagation();
+      }
+    };
+    document.addEventListener("focusin", handler);
```

Comment:
I mentioned this in slack, but Bootstrap 4 doesn't appear to work using native event listeners. See http://fiddle.tiny.cloud/gRgaab/207. As such we probably need to clarify this is Bootstrap 5 only. I'm not sure if you want to give a Bootstrap 4 example as well?

Reply:
Yep, two versions now. Here's a test page for the bootstrap 4 version:
https://codesandbox.io/s/tinymce-link-plugin-bootstrap-modal-4-scmp7?file=/src/index.js